### PR TITLE
Connect profile view with appointment list

### LINKED ISSUE #44: Connect profile view with appointments list
# REQUIREMENTS
Add button to redirect profile patient view to appointments list

# Acceptance Criteria
- [ ] redirect correctly form profile to appointments list

### PR DESCRIPTION
**Pull request by OmarRoBra**
# Description
What did you implemented/Why did you implemented this:
 - I added a button in the patient profile view to connect the appointment list with the view
  
  ## Screenshots 
If applicable, include screenshots of the results or screenshots that help to see changes
image

### CODE REVIEW

**Review by YilKanda — DISMISSED** [review]
I believe that this change could have been implemented in some other issue that was related to this, so as not to have a PR for just one button.

File: db/schema.rb [hunk 1]
```diff
@@ -15,6 +15,19 @@
...
+    t.boolean "pay", default: false, null: false
+    t.string "message"
+    t.bigint "patient_id"
+    t.bigint "psychologist_id"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["patient_id"], name: "index_appointment_on_patient_id"
+    t.index ["psychologist_id"], name: "index_appointment_on_psychologist_id"
+  end
+
```
Comment (dannharo) [inline db/schema.rb:30]: I think this is was added by mistake, this table was removed in this commit 
https://github.com/BC-MAY-21-ROR/proyecto-proy-t03/commit/c8d016ed7e2a72fec19170a29e7b25d6de9d38c6

File: app/views/home/profile_patient.erb [hunk 2]
```diff
@@ -8,7 +8,9 @@
         <div class="patient-info-about_txtarea">
           <%= current_patient.description %>
         </div>
+         <%= link_to t('My Appointments'), appointments_path, class:"btn btn-primary"%>
       </div>
+
```
Comment (dannharo) [inline app/views/home/profile_patient.erb:13]: remove this line

File: app/views/home/profile_patient.erb [hunk 3]
```diff
@@ -34,5 +36,6 @@
     </div>
 
     <%= link_to t("edit_profile"), edit_patient_registration_path, class:"buton_edit mb-2"%>
+
```
Comment (dannharo) [inline app/views/home/profile_patient.erb:39]: also this line

**Review by dannharo — CHANGES_REQUESTED** [review]
Check the comment on the `schema.rb`

**Review by brayandiazc — DISMISSED** [review]

**Pull request closed without merging**